src/rpc/rpcevo.cpp

```diff
@@ -353,7 +355,11 @@ UniValue protx_register(const JSONRPCRequest& request)
         ptx.vchSig.resize(65);
     }
 
-    FundSpecialTx(tx, ptx);
+    bool fUsePrivateSend = (request.params.size() > paramIdx + 6)
+        ? ParseBoolV(request.params[paramIdx + 6], "fUsePrivateSend")
```

Comment:
Shouldn't this be `usePrivateSend` instead of `fUsePrivateSend ` here?

Reply:
Fixed :)